client-src/elements/chromedash-feature-detail.js

```diff
@@ -388,6 +390,18 @@ class ChromedashFeatureDetail extends LitElement {
     return this.renderSection(summary, content);
   }
 
+  renderAddStageButton() {
+    if (!this.canEdit) {
+      return nothing;
+    }
+
+    return html`
+    <sl-button variant="primary" size="small" @click="${
```

Comment:
Let's not make it primary because adding a stage is an infrequent operation.

Reply:
Changed 👍